scripts/vesting.ts

```diff
@@ -0,0 +1,83 @@
+import BigNumber from "bignumber.js"
+import * as data from './data/rewards.json'
+
+type StaticVestingInfo = {
+    origin: string
+}
+
+type DynamicVestingInfo = {
+    destination: string,
+    schedule: {
+        amtToBeVestedOverTime: string,
+        start: string,
+        period: string,
+        per_period: string,
+        period_count: string 
+    }
+}
+
+type VestingBatch = {
+    statics: StaticVestingInfo
+    vestings: DynamicVestingInfo[]
+}
+
+const origin = '!!! address that pays bsx as they vest; treasury address I guess !!!' 
+// todo;  this is the address that pays out bsx as they vest, to target
+
+const thirtyPercentStart = '!!! end block of BSX LBP !!!'  
+// end block of BSX LBP, when is that ?
```

Comment:
when's this block ?

Reply:
We don't know yet, so for testing purposes pick a block yourself. We'll supply this down the line for the production run